README.md

```diff
@@ -59,26 +62,26 @@ Give an example
 Add additional notes about how to deploy this on a live system
 
 ## Built With
-
 * Ruby on Rails 5.2 - The web framework used
 
-
-## Contributing
+* Gems for testing
```

Comment:
Having this as a header was perfectly fine, and actually preferred that, it just that the header size was bigger then the `Built with` header,  and the `gems for testing` header should have been smaller then `Built with`